modules_extra/git.js

```diff
@@ -450,23 +448,53 @@ exports.create = function (api) {
 
       if(c.type === 'issue') {
         return h('div',
-          h('p', 'opened issue on ', repoLink(c.project)),
           c.title ? h('h4', c.title) : '',
           api.markdown(c)
         )
       }
 
       if(c.type === 'pull-request') {
         return h('div',
-          h('p', 'opened pull-request ',
-            'to ', repoLink(c.repo), ':', c.branch, ' ',
-            'from ', repoLink(c.head_repo), ':', c.head_branch),
           c.title ? h('h4', c.title) : '',
           api.markdown(c)
         )
       }
     },
 
+    message_title: function (msg) {
+      var c = msg.value.content
+
+      if(c.type === 'git-repo') {
+        return h('div', [
+          h('p', 'git repo ', repoName(msg.key)),
+          c.upstream ? h('p', 'fork of ', repoLink(c.upstream)) : ''
+        ])
+      }
+
+      if(c.type === 'git-update') {
+        return h('p', 'pushed to ', repoLink(c.repo))
+      }
+
+      if(c.type === 'issue-edit'
+       || (c.type === 'post' && c.text === '')) {
+        return h('div',
```

Comment:
lets use [ ] children everywhere for consistency and future compatability with mutant?